src/mono/wasm/wasm.proj

```diff
@@ -16,8 +16,8 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <MonoWasmThreads Condition="'$(WasmEnableThreads)' == 'true' or '$(WasmEnablePerfTracing)' == 'true' or '$(MonoWasmBuildVariant)' == 'multithread' or '$(MonoWasmBuildVariant)' == 'perftrace'">true</MonoWasmThreads>
-    <MonoWasmThreadsNoUser Condition="('$(WasmEnableThreads)' != 'true' and '$(WasmEnablePerfTracing)' == 'true') or '$(MonoWasmBuildVariant)' == 'perftrace'">true</MonoWasmThreadsNoUser>
+    <MonoWasmThreads Condition="'$(MonoWasmBuildVariant)' == 'multithread' or '$(MonoWasmBuildVariant)' == 'perftrace'">true</MonoWasmThreads>
```

Comment:
Just curious, why are we naming public properties as `WasmXYZ`, but these others as `MonoWasmXYZ`?

Reply:
> Just curious, why are we naming public properties as `WasmXYZ`, but these others as `MonoWasmXYZ`?

I think I wanted the public ones to have short memorable names.

For the internal ones I was using `MonoWasmXYZ` as a hint that it's only affecting how the runtime is configured at compilation time, and that it's not something user code can change later.